source/slang/slang-hash-utils.h

```diff
@@ -25,9 +28,9 @@ namespace Slang
     }
 
     // Returns the stored hash in checksum as a String.
-    inline StringBuilder hashToString(const slang::Digest& hash)
+    inline String hashToString(const Digest& hash)
```

Comment:
`digestToString`

Reply:
Changed to be a static class using the method naming scheme above